DtoImporterService/BookImporter.cs

```diff
@@ -21,7 +21,7 @@ protected override int DoImport(IEnumerable<ImportItem> importItems)
 			new CategoryImporter(DbContext).Import(importItems);
 
 			// get distinct
-			var productIds = importItems.Select(i => i.DtoItem.ProductId).ToList();
```

Comment:
In several places in this PR, you removed ToList(). Would these areas benefit from deferred execution?

Reply:
Probably not. It was just me imposing my style again, and for that I'm sorry!

Reply:
I understand the temptation. Use whichever convention you want on new code. I don't want to get into style wars though.